static-site/src/util/blogPosts.js

```diff
@@ -36,6 +42,76 @@ export function getBlogPosts() {
   return blogPosts;
 }
 
+/**
+ * Writes RSS2, Atom, and JSON feeds into the `public/blog` directory.
+ */
+export async function generateBlogFeeds() {
+  const feedDir = `${__dirname}/../../public/blog`;
+  const siteURL = "https://nextstrain.org/blog";
```

Comment:
```suggestion
  const blogURL = "https://nextstrain.org/blog";
```

I was confused why these paths seemed to look different:

https://github.com/nextstrain/nextstrain.org/blob/5709c3ce11f60cf66fa4ae3604643018e70e2e00/static-site/src/util/blogPosts.js#L62

https://github.com/nextstrain/nextstrain.org/blob/5709c3ce11f60cf66fa4ae3604643018e70e2e00/static-site/src/components/layout.jsx#L26

then I realized that `siteURL` and `siteUrl` are not the same. Renaming `siteURL` (or even better, [putting the Atom/JSON/RSS2 paths in shared variables](https://github.com/nextstrain/nextstrain.org/pull/990#discussion_r1731861727)) would reduce confusion.

Reply:
Kinda ended up doing both of these things...